vnext/ReactWindowsCore/Modules/UIManagerModule.cpp

```diff
@@ -388,6 +388,8 @@ void UIManager::measure(
     auto &currNode = m_nodeRegistry.getNode(rootTag);
     if (currNode.m_parent == -1)
       break;
+    if (currNode.m_isWindowedPopup)
```

Comment:
>if (currNode.m_isWindowedPopup) [](start = 4, length = 31)

Will the measure still need when m_isWindowedPopup=true? For example, resizing the popup itself.

Reply:
We still need the measure, but I'm going to move this from ReactWindowsCore and just do this in NativeUIManager per Andy's suggestion.

---
In reply to: [304626882](https://github.com/microsoft/react-native-windows/pull/2781#discussion_r304626882) [](ancestors = 304626882)